kagenti/installer/app/components/keycloak.py

```diff
@@ -171,6 +171,23 @@ def install():
         ],
         "Deploying Keycloak statefulset",
     )
+
+    # Configure Keycloak to start with dev mode
+    run_command(
+        [
+            "kubectl",
+            "patch",
+            "statefulset",
+            "keycloak",
+            "-n",
+            "keycloak",
+            "--type=json",
+            "-p",
+            '[{"op": "replace", "path": "/spec/template/spec/containers/0/args", "value":["start-dev"]}]',
+        ],
+        "Updating Keycloak statefulset to enable OAuth",
```

Comment:
This is where we automatically configure Keycloak to use `start-dev`.

However, eventually, we don't want to use developer mode for production? So I guess we need to document to remove this?

Reply:
We use `start-dev` because that allows HTTP in addition to HTTPS and relaxes host name checking.  Probably the Kagenti installer needs an option for this that is used when configuring Keycloak.  On OpenShift it would create Routes to terminate TLS, on Kind it would either use `start-dev` or install an NGINX to terminate TLS.

Reply:
So should we make this optional somehow? Configurable with an env var?

I was just wondering if we should document this as an issue. If we leave it in, it'd be a security vulnerability?

Reply:
The `uv run kagenti-installer` is responsible for installing Keycloak.  It should also be responsible for configuring TLS.  Instead of trying to handle security in this PR, just get it working.  Create a new Issue for someone to create installer options to terminate TLS.  The issue should point out that once TLS is terminated properly we can use `start` instead of `start-dev` and TLS endpoints for Keycloak.

Reply:
+1